debezium-core/src/main/java/io/debezium/pipeline/EventDispatcher.java

```diff
@@ -147,6 +147,10 @@ public EventDispatcher(CommonConnectorConfig connectorConfig, TopicSelector<T> t
                 .build();
     }
 
+    public void setNeverSkip(boolean neverSkip) {
```

Comment:
Instead of injecting the value, do you think that `disableOperationSkipping()` (which sets `this.neverSkip = true`) would be better from a DSL point of view?

Reply:
Yes, you are right. it makes sense and also it would be nicer. I'll change it tonight.